lldb/source/Host/common/TCPSocket.cpp

```diff
@@ -115,6 +115,18 @@ std::string TCPSocket::GetRemoteConnectionURI() const {
   return "";
 }
 
+std::vector<std::string> TCPSocket::GetListeningConnectionURI() const {
+  if (m_listen_sockets.empty())
+    return {};
+
+  std::vector<std::string> URIs;
+  for (auto &s : m_listen_sockets)
```

Comment:
```suggestion
  for (const auto &[fd, addr] : m_listen_sockets)
```

(or any other variable names you like)

Reply:
Applied suggestion